src/include/storage/storage_defs.h

```diff
@@ -209,6 +209,17 @@ class VarlenEntry {
    */
   const byte *Content() const { return content_; }
 
+  /**
+   * @param rhs is the right hand side of the comparison
+   * @return true iff this < rhs according to a lexicographic order.
+   */
+  bool operator<(const VarlenEntry &rhs) const {
```

Comment:
Not sure this is a good idea: not every varlen entry makes sense in a comparison (a blob, for example)